onedal/common/validation.py

```diff
@@ -126,7 +126,9 @@ def _check_X_y(X, y, dtype="numeric", accept_sparse=False, order=None, copy=Fals
     return X, y
 
 
-def _get_sample_weight(X, y, sample_weight, class_weight, classes, svm_type):
+def _get_sample_weight(X, y, sample_weight, class_weight, classes, svm_type, nu=0):

```

Comment:
move this function to the svm class, because it is not suitable for general use for now